systems/identity/.local.docs/task/025-application-api-auth-account-integration.md

```
@@ -0,0 +1,36 @@
+# 025-application-api-auth-account-integration
+
+## 작업 개요
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)